lib/algfld.gi

```diff
@@ -212,18 +212,38 @@ local f,p,nam,e,fam,colf;
 end;
 
 InstallMethod(AlgebraicExtension,"generic",true,
-  [IsField,IsUnivariatePolynomial],0,DoAlgebraicExt);
+  [IsField,IsUnivariatePolynomial],0,
+function(k,f) return DoAlgebraicExt(k,f,true);
+end);
```

Comment:
You could use `{k,f} -> DoAlgebraicExt(k,f,true)` here, which is slightly more concise.

Reply:
I'm aware of this new syntax. But I prefer to have code which is also readable with older GAP versions which I'm still using.